adcm_aio_client/core/actions/_objects.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import annotations
+
+from functools import cached_property
+from typing import TYPE_CHECKING, Any, Self
+
+from asyncstdlib import cached_property as async_cached_property
+
+from adcm_aio_client.core.errors import HostNotInClusterError, NoMappingRulesForActionError
+from adcm_aio_client.core.mapping import ActionMapping
+from adcm_aio_client.core.objects._accessors import NonPaginatedChildAccessor
+from adcm_aio_client.core.objects._base import InteractiveChildObject, InteractiveObject
+
+if TYPE_CHECKING:
+    from adcm_aio_client.core.objects.cm import Cluster
```

Comment:
why Cluster is imported twice?

Reply:
because it's imported for type checking (in this case there's no circular imports, I believe)